src/webgpu/idl/constructable.spec.ts

```diff
@@ -0,0 +1,56 @@
+export const description = `
+Test that constructable WebGPU objects are actually constructable.
+`;
+
+import { keysOf } from '../../common/util/data_tables.js';
+
+import { makeTestGroup } from './../../common/framework/test_group.js';
+import { IDLTest } from './idl_test.js';
+
+export const g = makeTestGroup(IDLTest);
+
+const errors = {
+  GPUInternalError: globalThis.GPUInternalError,
+  GPUOutOfMemoryError: globalThis.GPUOutOfMemoryError,
+  GPUValidationError: globalThis.GPUValidationError,
+};
+
+g.test('gpu_errors')
+  .desc('tests that GPUErrors are constructable')
+  .params(u => u.combine('errorType', keysOf(errors)))
+  .fn(t => {
+    const { errorType } = t.params;
+    const Ctor = errors[errorType];
```

Comment:
optional nit: I think could just be `globalThis[errorType]` and flatten `keysOf(errors)` down into an inline array.

Reply:
done.  I thought I tried that originally and got type issues but trying it again it's working. No idea what I did wrong before 😅